track-modules.sh

```diff
@@ -1,13 +1,40 @@
 #!/bin/bash
 
-deps(){
+F='/tmp/mod.txt'
```

Comment:
Using a static name for the temporary file causes concurrency issues: what if two track_modules commands are executed at the same time?

Check `mktemp` for a way to generate temporary files with unique names in scripts.

Reply:
I update using mktemp now! thanks